db/migrate/20180924082712_add_title_to_users.rb

```diff
@@ -0,0 +1,5 @@
+class AddTitleToUsers < ActiveRecord::Migration
+  def change
+    add_column :users, :title, :string
```

Comment:
re translation, we can do `title_en` and `title_zh_tw`

Reply:
Based on comments in GCW-2132 I'll retract this suggestion. @abulsayyad123 is correct - it doesn't really make sense to have title_en and title_zh_tw (consider first_name and last_name also)

Reply:
Thanks @steveyken . I have rollback migration for those fields.